Microsoft.Toolkit.Uwp.UI.Controls.Design/Controls/WrapPanel.Typedata.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
+{
+#if VS_DESIGNER_PROCESS_ISOLATION
```

Comment:
@Nirmal4G this is to support the old model, right? I don't think we don't need to do that. Double-checking with the VS team on if they think we should be supporting the old model still. If not, we can just drop support than trying to maintain this complexity. Will let you know shortly.

Reply:
> This change is for VS XAML Designer and not Windows revision specific. As long as devs don't need to hang on to the older VS versions we can safely remove this in the future.

[That said](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3318#issuecomment-706270164), I'll delete those in a new PR, so that we can revert it, if we ever need them again.

Reply:
@Nirmal4G yeah, checked with the VS team, sounds like later versions of VS 2017 should still load the new format, so we should just focus on the new DesignTools format to ship only to simplify the codebase. Sounds like the only thing that may be lost is seeing Icons in VS2017, but it'll be better to keep things simpler for the Toolkit to ship one thing for this space. Thanks!

Reply:
If we're doing a squash merge, it's better to keep this in the history. If not, I can add the changes here instead of creating a new PR!

@michael-hawker ?

Reply:
@Nirmal4G whatever works best for you, you can rebase off the main branch and force-push clean history here if you'd like as well. I think the main thing is we should just have the one PR that brings us from the old system to the new system, thanks!